spec/fixtures/katello-certs-check/missing-parameter.txt

```diff
@@ -2,4 +2,4 @@ One of the required parameters is missing.
 Verifies, that custom SSL certificate files are usable
 as part of the Katello installation. When passing filenames use absolute paths.
 
-usage: |COMMAND| -c CERT_FILE -k KEY_FILE -b CA_BUNDLE_FILE
+usage: |COMMAND| -t TARGET -c CERT_FILE -k KEY_FILE -b CA_BUNDLE_FILE
```

Comment:
This is not needed, this is just for Puppet related things, the error in the Bash script if a user misses a param is good enough.